cohort_schedules/C04_classroom.md

```diff
@@ -336,7 +336,7 @@ Week 11 brings the first group, multi-week project for students. This is a major
 In addition to all of this, students are learning how to perform user authentication in Rails using session data and controller filters. Advanced validations for model classes are also discussed, and the bEtsy project provides several opportunities for using custom validations.
 
 ### Learning Goals
-- Long-term project development with the Agile methodology 
+- Long-term project development with the Agile methodology
 - Different team member roles (task manager, stand up leader, project manager)
 - Password-based user authentication in Rails
 - Authorization of specific actions/routes in Rails
```

Comment:
This was something that was brought up in feedback: not being clear about what the goals of the capstone were/are. So just a note to us to remember to go over these goals with the students at the beginning (and then revisiting a couple times during capstone).

Also making sure that these are the goals that we really want.